msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -998,3 +998,4 @@ The following table provides the response codes:
 * [Apps for Teams meetings](teams-apps-in-meetings.md)
 * [Live Share SDK](teams-live-share-overview.md)
 * [Teams cloud meeting recording](/microsoftteams/cloud-recording)
+* [Get meetingAttendanceReport](/graph/api/meetingattendancereport-get)
```

Comment:
Do not use jargon, names of classes/methods/API calls/etc. in link text. Rewrite it based on the below descriptions from the linked articles.
"_Contains information associated with a meeting attendance report._" 
and 
"_Get the meetingAttendanceReport for an onlineMeeting. Each time an online meeting ends, an attendance report will be generated for that session._"